Library/Homebrew/test/cask/audit_spec.rb

```diff
@@ -5,7 +5,7 @@
 describe Cask::Audit, :cask do
   def include_msg?(problems, msg)
     if msg.is_a?(Regexp)
-      Array(problems).any? { |problem| problem[:message] =~ msg }
+      Array(problems).any? { |problem| problem[:message]&.match?(msg) }
```

Comment:
Haven't tracked this one but you can definitely assume `msg` is not nil and call this that way.